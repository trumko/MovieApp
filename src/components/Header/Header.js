import React from "react"
import classnames from 'classnames'

import Logo from 'components/Logo/Logo'

import './Header.scss'

export default function Header({ className, children }) {

  return (
    <div className={classnames('mr_header', className)}>
      <div className="mr_container mr_headerContent">
        <Logo />
        {children}
      </div>
    </div>
  )
}
