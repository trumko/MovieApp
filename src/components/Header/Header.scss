.mr_header {

}

.mr_headerContent {
  background: url(../../assets/movie-collage.jpg) 50% 50% no-repeat;
  background-size: cover;
  min-height: 170px;
}

h2 {
  color: #fff;
}
