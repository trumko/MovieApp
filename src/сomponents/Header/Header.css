.mr_header {

}


.mr_headerContent {
  background: url(../../assets/movie-collage.jpg) 50% 50% no-repeat;
  /* background: url(../../assets/netflix_header.jpg) 50% 50% no-repeat; */
  background-size: cover;
  min-height: 170px;
  filter: brightness(44%);
}
