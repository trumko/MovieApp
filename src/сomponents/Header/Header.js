import React, { Component } from "react";
import { Link } from 'react-router-dom';

import HomeHeader from './components/HomeHeader/HomeHeader';
import MovieHeader from './components/MovieHeader/MovieHeader';

import './Header.css'

export default class Header extends Component {
  render() {
    const { isHome } = this.props;
    return (
      <div className="mr_header">
        <div className="mr_container mr_headerContent">
          {isHome ? <HomeHeader /> : <MovieHeader />}
        </div>
      </div>
    )
  }
}
