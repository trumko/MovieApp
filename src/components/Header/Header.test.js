import React from 'react'
import { shallow } from 'enzyme'
import Header from './Header'

describe('Header', () => {
  it('should be rendered correctly with custom clasName', () => {
    const component = shallow(<Header className="testClass" />);

    expect(component).toMatchSnapshot();
  });
});
