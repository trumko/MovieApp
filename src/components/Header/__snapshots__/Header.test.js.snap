// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header should be rendered correctly with custom clasName 1`] = `
<div
  className="mr_header testClass"
>
  <div
    className="mr_container mr_headerContent"
  >
    <Logo />
  </div>
</div>
`;
